addons/transform/org.openhab.transform.jinja/src/main/java/org/openhab/transform/jinja/internal/JinjaTransformationService.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
```

Comment:
The Jenkins build is failing because of wrong license headers. Can you please run `mvn license:format` in the root directory of this project and come up with a new PR for your fixed headers? Thanks.

You mayb want to take care about the warning too.

```
[ERROR] Code Analysis Tool has found: 
 4 error(s)! 
 2 warning(s) 
 0 info(s)
[ERROR] org.openhab.transform.jinja.internal.profiles.JinjaTransformationProfileFactory.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2019 Contributors to the openHAB project$
[ERROR] org.openhab.transform.jinja.internal.profiles.JinjaTransformationProfile.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2019 Contributors to the openHAB project$
[ERROR] org.openhab.transform.jinja.internal.JinjaTransformationService.java:[2]
Header line doesn't match pattern ^ \* Copyright \(c\) 2010-2019 Contributors to the openHAB project$
[WARNING] .transform.jinja/ESH-INF/config/jinjaProfile.xml:[3]
There were whitespace characters used for indentation. Please use tab characters instead
[ERROR] .transform.jinja/ESH-INF/config/jinjaProfile.xml:[0]
The file ESH-INF/config/jinjaProfile.xml isnt included in the build.properties file. Good approach is to include all files by adding `ESH-INF/` value to the bin.includes property.
```

Reply:
Oh my word, I haven't caught that as well (probably because I'm also compiling without tests and checks at the moment). Should we revert this PR?

Reply:
> Should we revert this PR?

No, we should come up with a follow-up PR to fix it.

Reply:
Maybe it wasn't an issue if it the addon was reviewed by 2 maintainers:

> New add-on contributions should be reviewed by 2 maintainers - always feel free to ping `@kaikreuzer` to do a second/final approval.

There's now https://github.com/openhab/openhab2-addons/pull/4984

Reply:
True. I'm just eager to get the required parts in for full MQTT HomeAssistant support and I know that you guys are busy with the build system and IDE setup.